### ISSUE TITLE
lines stroke (color) legend shows circles

### ISSUE THREAD

**Issue by yasirs** [rstudio/ggvis#360]
The legend for lines should show lines, rather than circles (which look as if they represent markers for points). E.g., if I do

    mtcars %>% ggvis(~wt, ~mpg) %>% group_by(cyl) %>% layer_lines(stroke = ~factor(cyl))
I get 
![plot_276750617](https://cloud.githubusercontent.com/assets/400538/6152586/613e6134-b1ec-11e4-9611-bc00898633bb.png)
Also, I get a warning message:
> In rbind_all(out[[1]]) : Unequal factor levels: coercing to character

**Comment by raubreywhite** [comment]
Agreed. This is problematic for putting ggvis products into production.
